src/lib/openapi-to-json-schema.ts

```diff
@@ -238,12 +239,16 @@ function searchForExampleByPointer(pointer: string, examples: PrevSchemasType =
  * @see {@link https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.1.0.md#schemaObject}
  * @param data OpenAPI Schema Object to convert to pure JSON Schema.
  * @param opts Options
- * @param opts.addEnumsToDescriptions Whether or not to extend descriptions with a list of any present enums.
+ * @param opts.addEnumsToDescriptions Whether or not to extend descriptions with a list of any
+ *    present enums.
  * @param opts.currentLocation Current location within the schema -- this is a JSON pointer.
- * @param opts.globalDefaults Object containing a global set of defaults that we should apply to schemas that match it.
+ * @param opts.globalDefaults Object containing a global set of defaults that we should apply to
+ *    schemas that match it.
  * @param opts.isPolymorphicAllOfChild Is this schema the child of a polymorphic `allOf` schema?
- * @param opts.prevSchemas Array of parent schemas to utilize when attempting to path together examples.
+ * @param opts.prevSchemas Array of parent schemas to utilize when attempting to path together
+ *    examples.
  * @param opts.refLogger A function that's called anytime a (circular) `$ref` is found.
+ * @param opts.transformer A function that's called to potentially transform any discovered schema.
```

Comment:
This is a great idea but I'm going to do this in a followup PR because we have a lot of types set up like this right now where descriptions are in the docblock but not being pulled through in intellisense.

Reply:
![](https://media.tenor.com/t4LTqvGNTGUAAAAC/good-idea-i-stand-by-it.gif)